packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -227,6 +232,14 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox,
    * Note: This may increase the amount of time it takes for the comboBox itself to mount.
    */
   persistMenu?: boolean;
+
+  /**
+   * When specified, determines whether the callout (the menu which drops down) should
+   * restore the focus after being dismissed or not.  If false, then the menu will not try
+   * to set focus to whichever element had focus before the menu was opened.
+   * Defaults to true.

```

Comment:
Same as the above comment - to ensure that the default value shows up the the API references table, could you add a `@defaultvalue true` line to the TSDoc comment? Thanks!